shaped_reward_episodic_runner.py

```diff
@@ -93,11 +93,14 @@ def __call__(self, time_steps, sample=True, evaluation=False,
                 self.plan_grounded_atoms = self.g_utils.apply_grounded_plan(previous_state_grounded_atoms, self.plan)
             if evaluation:
                 print(env.envs[0].max_plan_step_reached)
-            next_ob, _, done, env_info = env.step(action)
+            next_ob, reward, done, env_info = env.step(action)
             next_state_grounded_atoms = self.g_utils.get_state_grounded_atoms(env.envs[0])
             if env.envs[0].reward_type == None:
                 reward, info = self.g_utils.get_shaped_reward(env.envs[0], next_ob, previous_state_grounded_atoms, next_state_grounded_atoms, self.plan_grounded_atoms)
                 reward = np.array([reward])
+            else:
+                info = self.g_utils.get_info(env.envs[0], next_ob)
```

Comment:
We dont need this I think since we can do with just env_info in this case?

Reply:
I was getting an error saying that `info` is not defined. We'll have to slightly refactor below line if we don't want to compute an info here